### DISCUSSION TITLE
Feature Request: separate tabs for repositories and fork

### DISCUSSION THREAD

**Discussion by kevinesaa** [community/community#53] [Discussions]
it would be great, when I see my profile or someone else's profile, watch the repositories created by the owner separated from the forks.

![imagen](https://user-images.githubusercontent.com/20506502/98430619-2ea16380-2085-11eb-8c28-dc8bebe49175.png)

**Comment by noraa-junker** [comment]
That's a feature for Github and not GitHub discussions

**Reply by kevinesaa** [reply]
Please, can you show me the link where I should post this?
